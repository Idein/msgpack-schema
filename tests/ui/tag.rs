use msgpack_schema::*;

#[derive(Serialize)]
struct S1 {
    x: String,
}

#[derive(Deserialize)]
struct S2 {
    x: String,
}

#[derive(Serialize)]
enum E1 {
    V,
}

#[derive(Deserialize)]
enum E2 {
    V,
}

fn main() {}
