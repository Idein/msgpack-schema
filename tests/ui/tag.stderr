error: no #[tag] given
 --> $DIR/tag.rs:5:5
  |
5 |     x: String,
  |     ^^^^^^^^^

error: no #[tag] given
  --> $DIR/tag.rs:10:5
   |
10 |     x: String,
   |     ^^^^^^^^^

error: no #[tag] given
  --> $DIR/tag.rs:15:5
   |
15 |     V,
   |     ^

error: no #[tag] given
  --> $DIR/tag.rs:20:5
   |
20 |     V,
   |     ^
